docs/alerting/monitors.md

```diff
@@ -32,7 +32,7 @@ Destination | A reusable location for an action, such as Amazon Chime, Slack, or
 
 1. Choose **Alerting**, **Destinations**, **Add destination**.
 1. Specify a name for the destination so that you can identify it later.
-1. For **Type**, choose Slack, Amazon Chime, or custom webhook.
+1. For **Type**, choose Slack, Amazon Chime, custom webhook, or [email](#email-as-a-destination).
 1. Specify the webhook URL. These requests use the HTTP POST method.
```

Comment:
Options here change depending on what you selected.

Reply:
Do you want me to add this sentence to the line that has the options?

Reply:
To add to Andrew's comment, maybe something like:

`"For the Email type, refer to the 'Email as a destination' section below. For all other types, specify the webhook URL."`.

Or remove this line and link to the appropriate part of the documentation for each type like you did for email above.